sysutils/apcupsd/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+
+name                        apcupsd
+version                     3.14.14
+categories                  sysutils
+platforms                   darwin
+license                     GPL-2
+maintainers                 stephenreay
```

Comment:
as shown in the guide and/or other Portfiles, one should list here a GitHub handle, including the "@" and an obfuscated e-mil address. These should be within curly brackets to signify that the belong to the same person. Finally, ideally "openmaintainer" should be added to allow others to make minor changes to the Portfile without your explicit permission, unless there is a very compelling reason not to add this.